packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -15,20 +15,20 @@
 import {Api} from '../../src/Api';
 import staticMetadata from '../../src/staticMetadata';
 import WsProvider from '@plugnet/rpc-provider/ws';
-import {Metadata} from '@plugnet/types';
+import {Metadata, Hash} from '@plugnet/types';
 
 describe('e2e api create', () => {
-    it('For Kauri environment - checking if static metadata is same as latest', async () => {
+    it.skip('For Kauri environment - checking if static metadata is same as latest', async () => {
```

Comment:
can enable these two tests

Reply:
do we still want to keep this test for kauri here? I mean, Kauri is not public.

Reply:
ok, will remove it in another PR